101-azure-api-for-fhir/azuredeploy.json

```diff
@@ -0,0 +1,56 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "serviceName": {
+            "type": "string",
+            "metadata": {
+                "description": "The name of the service."
+            }
+        },
+        "environmentName": {
```

Comment:
use the environment() function to get the env for deployment - but if it's not available in Azure Gov, just remove this.